aleph0/knowledge/nextjs13/general.txt

```diff
@@ -25,3 +25,16 @@ Next.js uses a file-system based router where folders are used to define routes.
 Each folder represents a route segment that maps to a URL segment. To create a nested route, you can nest folders inside each other.
 A Dynamic Segment can be created by wrapping a folder's name in square brackets: [folderName]. For example, [id] or [slug].
 
+Notes
+DO NOT update files in this path (components/ui/...), only create new ones.
```

Comment:
Will be a challenge if we're expecting users to always be correcting and wrestling with the ai. Not sure there's a better way though.

Reply:
This is something specific to our nextjs package, no?

Reply:
Any project could have specific rules like this? This rule specifically is about shadcn that places components in the components/ui folder. On another project it could just be not to replace any existing component files.